service/controller/resource/tccp/testdata/case-0-basic-test-route53-enabled.golden

```diff
@@ -133,7 +133,7 @@ Resources:
       HealthCheck:
         HealthyThreshold: 2
         Interval: 5
-        Target: TCP:2379
+        Target: HTTP:2379/healthz
```

Comment:
This does not look right. I feel like I misunderstand something here. How many health checks do we have and which endpoints/ports do they check against?

Reply:
Oh we use the same function for both api and etcd, dammit

Reply:
fixed